### DISCUSSION TITLE
File managment

### DISCUSSION THREAD

**Discussion by priyansh004** [excalidraw/excalidraw#6829] [General]
This website needs new file saving structure or file management in chrome it self .
is there any extension of this which i am not aware of ?

**Comment by ad1992** [comment]
@priyansh004 can you elaborate more about the use case ?

**Reply by priyansh004** [reply]
Like adding the file management folder which include the different folder for storing the file in drive ( maybe ). And adding feature like sharing folder of different files or collaboration with other user. And also better UI.
